tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/wizard/NewLiferayPluginProjectWizard.java

```diff
@@ -174,6 +206,7 @@ protected void performPostFinish()
         {
             new WorkspaceJob( "Configuring project with Ivy dependencies" ) //$NON-NLS-1$
             {
+
```

Comment:
No need for this change, the formatter seems to have been applied to files that didn't change, so lets not include that as a change